src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -490,6 +490,86 @@
                 }
             ]
         },
+        {
+            "path": "/v1/debug/partitions/{namespace}/{topic}/{partition}/block_raft_op/{op}",
```

Comment:
when looking at this it seems to be that it would be better to use honebadger, the mechanism is already there